Gopkg.lock

```diff
@@ -136,6 +136,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "6ade71befc0ce2335f0dc7f11d68930ccaffee771dfdc49f92673dea0b564cc8"
+  inputs-digest = "158b0077955d6d252f0b332b791bc8fb744b7620871ee86018e115c41e111dc6"
```

Comment:
Do we need this specifically for this change? Or is this something that we should fix on the `nextercism` branch and then rebase this PR onto that?